app/models/post.rb

```diff
@@ -0,0 +1,27 @@
+class Post < ApplicationRecord
+  # associations
+  belongs_to :author, class_name: 'User'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  # callback
+  after_save :update_user_posts_counter
+  after_initialize :set_default
+
+  # methods
+
+  def five_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_user_posts_counter
+    author.update(post_counter: author.posts.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Have you ever considered using the `increment` Ruby method?

__HINT:__ 
```Ruby
def update_user_posts_counter
  author.increment!(:posts_counter)
end
```